p2-maven-plugin/src/main/java/org/eclipse/tycho/p2maven/transport/RemoteArtifactRepositoryManagerAgentFactory.java

```diff
@@ -34,6 +35,9 @@ public class RemoteArtifactRepositoryManagerAgentFactory implements IAgentServic
 	@Requirement
 	MavenAuthenticator authenticator;
 
+	@Requirement
+	MavenContext mavenContext;
```

Comment:
please use `LegacySupport` to access the session directly to get its properties

Reply:
Done